src/reflect/scala/reflect/internal/util/Statistics.scala

```diff
@@ -267,19 +308,20 @@ quant)
    */
   final val hotEnabled = false
 
-  def enabled = _enabled
+  @inline def enabled = canEnable
   def enabled_=(cond: Boolean) = {
-    if (cond && !_enabled) {
+    if (cond && !canEnable) {
       val start = System.nanoTime()
       var total = 0L
       for (i <- 1 to 10000) {
         val time = System.nanoTime()
         total += System.nanoTime() - time
       }
       val total2 = System.nanoTime() - start
+      // Shouldn't we replace this println with a `reporter.info`?
```

Comment:
I guess this is a TODO? I'd like to get some confirmation. Problem is that we have no access to reporters from here.